lib/burndown_chart.rb

```diff
@@ -160,6 +160,7 @@ def process_options(hash)
         cli_switches << "--output #{hash['output']}" if hash['output']
         cli_switches << '--verbose'                  if hash['verbose']
         cli_switches << "--location #{hash['location']}" if hash['location']
+        cli_switches << '--no-head' if hash['no-head']
```

Comment:
you shouldn't modify the code to be able to write a test 😱

Reply:
Sorry, actually I thought this could be implemented as a option and didnt realize that it had to be passed directly to python scripts for automation.